packages/storage/amplify_storage_s3/test/amplify_storage_s3_impl_test.dart

```diff
@@ -21,7 +21,8 @@ void main() {
           ),
         },
       ),
-    );
+      // ignore: invalid_use_of_internal_member
+    ).toAmplifyOutputs();
```

Comment:
Similar to other comments - Should we remove `AmplifyConfig` completely from the tests?